CaesarCipher.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+public class CaesarCipher {
+    public static String caesarCipher(String input, int[] shiftPattern) {

```

Comment:
define every method in brief using comments with short definition , input parameters and return values.